libs/expo/betterangels/src/lib/screens/AddEditClient/RelevantContacts/Contact/index.tsx

```diff
@@ -1,4 +1,9 @@
-import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import {
```

Comment:
Hi @Davit-BetterAngels 

2 things.

1. I mentioned this to Dave for [Limit digits of phone number while adding/editing client with error message (DEV-435)](https://github.com/BetterAngelsLA/monorepo/pull/549) but just wanted to let you know I don't see the validation message when:

- It's 10 digits but the area code does not suit for US phone number. e.g. 1112223333 

But I guess you can leave it and wait for him to fix it.

2. I don't see the validation for Email address "Enter a valid email address" when I entered only the name of the person for example.

<img src="https://github.com/user-attachments/assets/2db81a2c-49c9-42fd-b6f4-938a2a917e74" width="360" />

<img src="https://github.com/user-attachments/assets/61f3ae94-6dc7-4d28-8a89-4e2a302c3bec" width="360" />

Reply:
> Hi @Davit-BetterAngels
> 
> 3 things.
> 
> 1. I mentioned this to Dave for [Limit digits of phone number while adding/editing client with error message (DEV-435)](https://github.com/BetterAngelsLA/monorepo/pull/549) but just wanted to let you know I don't see the validation message when:
> 
> * It's 10 digits but the area code does not suit for US phone number. e.g. 1112223333
> 
> But I guess you can leave it and wait for him to fix it.
> 
> 2. I don't see the validation for Email address "Enter a valid email address" when I entered only the name of the person for example.
> 3. I didn't see the ZIP code section in Contact Info.
> 
> <img alt="" width="360" src="https://private-user-images.githubusercontent.com/126514913/373032736-2db81a2c-49c9-42fd-b6f4-938a2a917e74.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MDM1MTAsIm5iZiI6MTcyNzkwMzIxMCwicGF0aCI6Ii8xMjY1MTQ5MTMvMzczMDMyNzM2LTJkYjgxYTJjLTQ5YzktNDJmZC1iNmY0LTkzOGEyYTkxN2U3NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQyMTA2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MmI5OGFmYTMzYzYyNjNlYjYyMWM0MmQ5OTc1ZWEzMWE3MGNmNDdhM2NjYTBlMTJiZWFhY2Y2OWE3NmVmNjE5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hfJV9Z_es8hZQRl4RhABYptXiBiphXrNuvtcMPXlmrM"> <img alt="" width="360" src="https://private-user-images.githubusercontent.com/126514913/373033458-61f3ae94-6dc7-4d28-8a89-4e2a302c3bec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MDM1MTAsIm5iZiI6MTcyNzkwMzIxMCwicGF0aCI6Ii8xMjY1MTQ5MTMvMzczMDMzNDU4LTYxZjNhZTk0LTZkYzctNGQyOC04YTg5LTRlMmEzMDJjM2JlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQyMTA2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00Njg3MTlmMTEwODZmMTBkZmVhZmMwMzVjYTc2YmMwNzNkZThkNjVkMjM3ZjI4OTg2OWI1ZTNiZWU0MGIzOTg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.reMO9bZAfBeLiildujXTHuGfOQzwII0pGAxzbAoYgnw"> <img alt="" width="360" src="https://private-user-images.githubusercontent.com/126514913/373034077-8500163e-7116-4921-bc5c-a34f9c05334b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MDM1MTAsIm5iZiI6MTcyNzkwMzIxMCwicGF0aCI6Ii8xMjY1MTQ5MTMvMzczMDM0MDc3LTg1MDAxNjNlLTcxMTYtNDkyMS1iYzVjLWEzNGY5YzA1MzM0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQyMTA2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYTJhMTJlNGRkYzFhYmFmN2FjOTY3MGMyMzYxZTFmZDQ4ZDliZmNhMzY3Mzg4OWY1NjUzY2FhMzI3Y2UyN2JiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-BDXwWP1f_ec87w5h_7-j8fsd-JXrOBljhrL07E7FQs">

1. phone number - nothing in AC says area code should match US area codes
2. email validation - mailing address is not email address. 
3. contact info - section is not implemented.

Reply:
@Davit-BetterAngels, re: 1 - it's not explicitly stated in AC, but it is strongly implied since phone numbers can't begin with a 1. I could be wrong, but I don't think any country has phone numbers that begin with 1

@shiozakiyuko, re: 3 - we're not going to have a zip-code field. 
- [PRD](https://betterangels.atlassian.net/wiki/spaces/DEV/pages/26640394/Client+Page+PRD#Contact-Information)
>  <img width="334" alt="image" src="https://github.com/user-attachments/assets/00c8e6b9-b0d3-429c-a476-3e77dbf1d3ed">
- [ticket](https://betterangels.atlassian.net/browse/DEV-755)
> <img width="424" alt="image" src="https://github.com/user-attachments/assets/13101796-1ac8-4de0-b59c-f027e17628fc">

Reply:
Thanks, @mikefeldberg and @Davit-BetterAngels.

As for 1, I thought you guys are using a Phone number component or something that you can universally use and comes with the validation (I was imagining that you are using what Dave created with the validation message). So let me know how you are using it but I didn't think you are implementing another phone number component for this user story... That's how I understood from Davit's comment... 

As for 2, when I looked at the ticket, I see Email Address in the AC (not mailing address). That's what I was testing and I didn't get to see the validation message when I submit a wrong format on purpose. 

Aso for 3, ok, thanks but it still shows as one of the AC... 

This is the validation ticket, correct? You guys are sending me with screenshots of different PR.. (PR584). lol Let me know if I'm misunderstanding here. 😕

Reply:
@Davit-BetterAngels and @mikefeldberg 

My apology for No.3 comment. I realized we are not doing the Street Address / City / State / Zip like the mockup on the ticket [DEV-655](https://betterangels.atlassian.net/browse/DEV-655). I understood what you guys mean now. So pass this through please.  😅 Thanks!